### PR TITLE
Add changelog of v2.27.0

### PR DESCRIPTION
**Pull request by RangiLyu**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: README_zh-CN.md [hunk 1]
```diff
@@ -76,22 +76,39 @@ MMDetection 是一个基于 PyTorch 的目标检测开源工具箱。它是 [Ope
...
 ### 🌟 3.x 预览版本
 
-全新的 **v3.0.0rc0** 版本已经在 2022.8.31 发布：
+#### 亮点
 
-- 基于 [MMEngine](https://github.com/open-mmlab/mmengine) 统一了各组件接口。
-- 全面支持混合精度，训练测试速度更快。
-- 提供了更强的基线模型，并支持了通用的半监督目标检测框架，详见[半监督目标检测教程](https://mmdetection.readthedocs.io/zh_CN/v3.0.0rc0/user_guides/semi_det.html)。
-- 支持使用任意单阶段检测器作为二阶段模型的 RPN，详见[教程](https://mmdetection.readthedocs.io/en/v3.0.0rc0/user_guides/single_stage_as_rpn.html)。
+我们很高兴向大家介绍我们在实时目标识别任务方面的最新成果 RTMDet，包含了一系列的全卷积单阶段检测模型。 RTMDet 不仅在从 tiny 到 extra-large 尺寸的目标检测模型上上实现了最佳的参数量和精度的平衡，而且在实时实例分割和旋转目标检测任务上取得了最先进的成果。 更多细节请参阅[技术报告](https://arxiv.org/abs/2212.07784)。 预训练模型可以在[这里](https://github.com/open-mmlab/mmdetection/tree/3.x/configs/rtmdet)找到。
```
Comment (hhaAndroid) [inline README_zh-CN.md:91]:
```suggestion
我们很高兴向大家介绍我们在实时目标识别任务方面的最新成果 RTMDet，包含了一系列的全卷积单阶段检测模型。 RTMDet 不仅在从 tiny 到 extra-large 尺寸的目标检测模型上实现了最佳的参数量和精度的平衡，而且在实时实例分割和旋转目标检测任务上取得了最先进的成果。 更多细节请参阅[技术报告](https://arxiv.org/abs/2212.07784)。 预训练模型可以在[这里](https://github.com/open-mmlab/mmdetection/tree/3.x/configs/rtmdet)找到。
```

**Review by hhaAndroid — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -77,23 +77,39 @@ Apart from MMDetection, we also released a library [mmcv](https://github.com/ope
...
+| ------------------------ | ------- | ------------------------------------ | ---------------------- |
+| Object Detection         | COCO    | 52.8                                 | 322                    |
+| Instance Segmentation    | COCO    | 44.6                                 | 188                    |
+| Rotated Object Detection | DOTA    | 78.9(single-scale)/81.3(multi-scale) | 121                    |
+
+<div align=center>
+<img src="https://user-images.githubusercontent.com/12907710/208044554-1e8de6b5-48d8-44e4-a7b5-75076c7ebb71.png"/>
+</div>
+
+A brand new version of **MMDetection v3.0.0rc5** was released in 26/12/2022:
```
Comment (BIGWangYuDong) [inline README.md:108]: released in 27/12/2022

File: README.md [hunk 3]
```diff
@@ -77,23 +77,39 @@ Apart from MMDetection, we also released a library [mmcv](https://github.com/ope
...
+
+- Support [RTMDet](https://arxiv.org/abs/2212.07784) instance segmentation models.
```
Comment (BIGWangYuDong) [inline README.md:110]: may need sync with https://github.com/open-mmlab/mmdetection/pull/9532

File: README_zh-CN.md [hunk 4]
```diff
@@ -76,22 +76,39 @@ MMDetection 是一个基于 PyTorch 的目标检测开源工具箱。它是 [Ope
 
 ### 💎 稳定版本
 
-最新的 **2.26.0** 版本已经在 2022.11.23 发布:
+最新的 **2.27.0** 版本已经在 2022.12.26 发布:
```
Comment (BIGWangYuDong) [inline README_zh-CN.md:79]: same

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**